client/src/pages/course/admin/events.tsx

```diff
@@ -224,16 +224,15 @@ function getColumns(handleEditItem: any, handleDeleteItem: any, { timeZone, even
   ];
 }
 
-function createRecord(values: any, courseId: number) {
+function createRecord(values: FormData) {
   const data = {
-    courseId,
     place: values.place,
     dateTime: values.dateTime ? formatTimezoneToUTC(values.dateTime, values.timeZone) : undefined,
     eventId: values.eventId,
     comment: values.comment,
 
     coordinator: values.coordinator,
-    organizerId: values.organizerId || null,
+    organizerId: values.organizerId || undefined,
```

Comment:
??

Reply:
the code allows passing empty string, which we do not want to propagate further. so better to use existing option.